test/resources/device_config.yml

```diff
@@ -95,4 +140,4 @@ DEVICES:
             host: proxy_pc_host
             login: proxy_pc_login
             password: password
-            set_timeout: False
+            set_timeout: False
```

Comment:
Please change set_timeot into `set_timeout: none`. False is a mistake but it works because of current implementation (`if set_timeout:`)